requirements/docker.txt

```diff
@@ -0,0 +1,13 @@
+-r pip.txt
+# http://initd.org/psycopg/docs/install.html#binary-install-from-pypi
+psycopg2-binary==2.8.3
```

Comment:
Thanks for the suggestion! I'm using -binary here just for simplicity and to avoid installing all the required packages into the docker image. We will be using this only for development at the moment.